package.json

```diff
@@ -39,5 +39,8 @@
   "bugs": {
     "url": "https://github.com/LePetitBloc/wallets/issues"
   },
-  "homepage": "https://github.com/LePetitBloc/wallets#readme"
+  "homepage": "https://github.com/LePetitBloc/wallets#readme",
+  "dependencies": {
```

Comment:
The wallets.json package should come with 0 dependencies when integrated with other projects.
At first I thought about the wallets update automation as a separate project.
But maybe it could live inside this very project, but all dependencies should be considered as dev then